### PR TITLE
Small Operator Docs improvements

### PR DESCRIPTION
**Pull request by tjanez**

### CODE REVIEW

File: src/operators/prerequisites.md [hunk 1]
```diff
@@ -10,12 +10,17 @@ operations.
 
 ### Downloading the binary
 
-For convenience we provide binaries that have been built by Oasis Labs. It is
-suggested, however, that you build from source yourself in a production
-deployment.
+::: tip
```
Comment (ravenac95) [inline src/operators/prerequisites.md]:
```suggestion
::: tip NOTE
```

File: src/operators/prerequisites.md [hunk 2]
```diff
@@ -33,12 +38,15 @@ params].
...
 
-### Using inside docker
+### Using Docker image
+
+::: warning DEPRECATED
+While possible to use, we no longer suggest nor do support usage of the
+Oasis-provided Docker images.
+:::
+
+For those that wish to use the Oasis-provided Docker image, the `oasis-node`
```
Comment (peterjgilbert) [inline src/operators/prerequisites.md]:
```suggestion
For those who wish to use the Oasis-provided Docker image, the `oasis-node`
```

File: src/operators/prerequisites.md [hunk 3]
```diff
@@ -33,12 +38,15 @@ params].
 
 [core-build]: https://github.com/oasislabs/oasis-core#developing-and-building-the-system
 [`master` branch]: https://github.com/oasislabs/oasis-core/tree/master/
-[params]: ./current-testnet-parameters.md
 
-### Using inside docker
+### Using Docker image
+
+::: warning DEPRECATED
+While possible to use, we no longer suggest nor do support usage of the
```
Comment (peterjgilbert) [inline src/operators/prerequisites.md]:
```suggestion
We no longer recommend or support using Oasis-provided Docker images.
```

File: src/operators/prerequisites.md [hunk 4]
```diff
@@ -33,12 +38,15 @@ params].
...
+Oasis-provided Docker images.
```
Comment (peterjgilbert) [inline src/operators/prerequisites.md]:
```suggestion
```

**Comment by tjanez** [comment]
Thanks for the review, I've updated the docs.

**Pull request merged**